Makefile

```diff
@@ -20,15 +21,24 @@ all: index-build index-push
 index-build: opm
 	@{ \
 	set -e ;\
+	mkdir -p $(BUILD_PATH)/$(INDEX_NAME) ;\
+	cp "$(INDEX_NAME).Dockerfile" $(BUILD_PATH)/ ;\
 	source ./$(OPERATORS_LIST) ;\
-	BUNDLES_DIGESTS='' ;\
 	for OPERATOR in $${OPERATORS[@]}; do \
-    	operator_bundle=$${OPERATOR/:*}-bundle ;\
-    	operator_version=$${OPERATOR/*:} ;\
-    	operator_digest=$$(skopeo inspect docker://$(REGISTRY)/$(ORG)/$${operator_bundle}:$${operator_version} | jq -r '.Digest') ;\
-	BUNDLES_DIGESTS+=$(REGISTRY)/$(ORG)/$${operator_bundle}@$${operator_digest}, ;\
+		operator_bundle=$${OPERATOR/:*}-bundle ;\
+		operator_version=$${OPERATOR/*:} ;\
+		operator_name=$${OPERATOR/:*} ;\
+		operator_digest=$$(skopeo inspect docker://$(REGISTRY)/$(ORG)/$${operator_bundle}:$${operator_version} | jq -r '.Digest') ;\
+		bundle_digest=$(REGISTRY)/$(ORG)/$${operator_bundle}@$${operator_digest} ;\
+		default_channel=$$(podman inspect $${bundle_digest} | jq -r '.[].Labels."operators.operatorframework.io.bundle.channel.default.v1"') ;\
+		channel="---\nschema: olm.channel\npackage: $${operator_name}\nname: $${default_channel}\nentries:\n  - name: $${operator_name}.$${operator_version}" ;\
+		$(OPM) init $${operator_name} --default-channel=$${default_channel} --output=yaml >> $(BUILD_PATH)/$(INDEX_NAME)/index.yml ;\
+		$(OPM) render $${bundle_digest} --output=yaml >> $(BUILD_PATH)/$(INDEX_NAME)/index.yml ;\
+		echo -e $${channel} >> $(BUILD_PATH)/$(INDEX_NAME)/index.yml ;\
 	done ;\
-	$(OPM) index add --bundles $${BUNDLES_DIGESTS%,} --tag $(INDEX_IMG) ;\
+	$(OPM) validate $(BUILD_PATH)/$(INDEX_NAME) ;\
+	podman build $(BUILD_PATH) -f $(BUILD_PATH)/$(INDEX_NAME).Dockerfile -t $(INDEX_IMG) ;\
```

Comment:
If a recall correctly, we added BUILDAH_FORMAT=docker to force the index schema version to be 2.x because the base image was v1. In this case, the base image (upstream-opm-builder) is already v2. I added the buildah_forman just in case.